packages/instrumentation-sequelize/src/types.ts

```diff
@@ -1,9 +1,21 @@
 import { Span } from '@opentelemetry/api';
 import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import { Sequelize } from 'sequelize';
```

Comment:
Can you please add the `type` here for the import?
```js
import type { Sequelize } from 'sequelize';
```

The `sequelize` package is only guaranteed to be available during transpiling, so instrumentation library is only allowed to use types from it.